app/src/main/java/com/connorlay/knitgrid/fragments/PatternDetailFragment.java

```diff
@@ -41,6 +46,8 @@
 
     private PatternPresenter mPatternPresenter;
 
+    private RowColumnListener mListener;
+
     public static PatternDetailFragment newInstance(PatternPresenter patternPresenter) {
```

Comment:
We should talk more about this. I'm gonna leave it as is for this PR, and then we can rethink the design after we've had a chance to discuss what it should look like.